blocks/footer/footer.css

```diff
@@ -1,14 +1,86 @@
-footer {
-  padding: 2rem;
-  background-color: var(--overlay-background-color);
+.footer {
+  font-size: var(--body-font-size-xs);
+}
+
+.footer a {
+  color: var(--text-color);
+}
+
+.footer ul {
+  display: flex;
+  flex-wrap: wrap;
+  list-style-type: none;
+}
+
+.footer h4 {
+  font-size: 18px;
+  font-family: var(--body-font-family);
+  color: var(--text-color);
+}
+
+.footer .center {
+  background-color: var(--highlight-footer-background-color);
+  padding: 54px 15px;
+}
+
+.footer .center input {
+  background: #fff;
+  border: 1px solid #aea780;
```

Comment:
Let's update this with variable name once we make some progress.

Reply:
> Let's update this with variable name once we make some progress.

yes